experimental/FTheoryTools/src/G4Fluxes/properties.jl

```diff
@@ -63,7 +63,7 @@ end
 @doc raw"""
     passes_verticality_checks(gf::G4Flux)
 
-G4-fluxes are subject to verticality conditions described in [Lin16](@cite).
+G4-fluxes are subject to verticality conditions described in [Lin16](@cite) and [KMW12](@cite).
```

Comment:
I would suggest to link to Timo Weigand F-theory TASI lectures instead. Maybe in fact, replace those two references with this?

Reply:
@apturner also suggests to reference instead of Ling's thesis the historically first paper that ever talked about these conditions. For this, just check which papers the above Tasi lectures reference.